build/Makefile.lnx

```diff
@@ -0,0 +1,6 @@
+BINARY_SERVER=alm
+BINARY_CLIENT=alm-cli
+FIND=find
+DBHOST=localhost

```

Comment:
If the DB host doesn't publish it's port to the docker host, the IP/DN can be different to `localhost`. But this is just a potential problem...